### ISSUE TITLE
[Bug] negative density for liquid fraction of a mixed phase stream

### ISSUE THREAD

**Issue by Fralopez** [DanWBR/dwsim6#212]
I have found that a mixed phase stream has negative density (and volumetric flow rate) for the liquid fraction.
The property package is Peng Robinson.
The same stream definition with the same property package is 100% vapour in Hysys, instead of 87%.

Conditions are:
T = 171 °C
P = 74.4 bar
mass flow = 530.7 kg/h

the stream contains H2, CO, CO2, H2O and CH4 (with zero N2 and O2, that are among the selected compounds)

**Comment by DanWBR** [comment]
Please follow the instructions from the bug reporting template. Your description doesn't allow me to reproduce this issue, so it will be ignored until you provide more information about it.

**Comment by Fralopez** [comment]
It is a simple flowsheet with a single stream. It was meant to find the stream properties. As I mentioned, I got negative liquid density. I then checked the same composition and conditions in Hysys and I got a vapor fraction = 1. So, apart from the negative density, maybe the liquid should not be there at all.

OS = Windows
Version = 6.6.3

Selected compounds = H2, CO, CO2, H2O, CH4, N2, O2
Property package = Peng Robinson

![0](https://user-images.githubusercontent.com/45826260/137016819-94a17b48-6aed-4442-af9a-dcfc4c5e143c.png)

![1](https://user-images.githubusercontent.com/45826260/137016828-40242413-817c-4502-b13e-7752c57fb5ba.png)

![2](https://user-images.githubusercontent.com/45826260/137016835-53bdeda5-62b0-47f1-bbc5-3ec14e3a7c76.png)
